backend/app/usecase/instrumentation/instrumentation.go

```diff
@@ -91,6 +94,29 @@ func (i Instrumentation) FeatureToggleRetrievalFailed(err error) {
 	}()
 }
 
+// SearchSucceed tracks the successes when searching the short resources.
+func (i Instrumentation) SearchSucceed(keywords string, t time.Time) {
+	go func() {
+		c := <-i.searchSucceedCh
+		i.metrics.Count("search-succeed", 1, 1, c)
+		userID := i.getUserID(nil)
+		props := map[string]string{
+			"keywords": keywords,
+			"time":     t.String(),
+		}
+		i.analytics.Track("Search", props, userID, c)
```

Comment:
@byliuyang can you please check why tracking location is not working? You can see in Segment dashboard that the city field is empty also for MadeFeatureDecision event. However, the keywords field is working fine.

My guess is that the proxy server is not correct. It seems that header X-Forwarded-For should be set manually from client in order to work.

Reply:
@arberiii It doesn't work locally. We have to test it on testing branch. Sorry about this.

Reply:
No worries, thank you for info!